src/composable/Multiplexer.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Multiplexer (ComposableCoW)', () => {
 
   test('constructor: cannot have undefined root for orders', () => {
     expect(() => {
-      const twap = Twap.default(generateRandomTWAPData())
+      const twap = Twap.fromData(generateRandomTWAPData())
```

Comment:
I change this method. Maybe we want to add in the future other ways to construct a new TWAP (i.e from the hex of the static input, or from the struct)

Reply:
Sure!